blockchain/config.go

```diff
@@ -76,6 +76,8 @@ type (
 		PersistStakingPatchBlock uint64 `yaml:"persistStakingPatchBlock"`
 		// FactoryDBType is the type of factory db
 		FactoryDBType string `yaml:"factoryDBType"`
+		// MintTimeout is the timeout for minting
+		MintTimeout time.Duration `yaml:"-"`
```

Comment:
It's not configurable by yaml. Make it in config is to disable in e2etest, avoiding many failures